.gitignore

```diff
@@ -66,6 +66,9 @@ node_modules/
 codekit-config.json
 .pycharm_helpers/
 
+.project
+.pydevproject
```

Comment:
Check with Carson, but I thought that we were keeping IDE specific gitignore settings in our local global gitignore configurations.